pkgs/by-name/cs/csharpier/package.nix

```diff
@@ -0,0 +1,19 @@
+{ buildDotnetGlobalTool, lib }:
+
+buildDotnetGlobalTool {
+  pname = "csharpier";
+  version = "0.26.4";
```

Comment:
[`0.27.0`](https://github.com/belav/csharpier/releases/tag/0.27.0) seems to have just released, if you'd like to bump before merge.